diff --git a/node_modules/react-devtools-core/dist/backend.js b/node_modules/react-devtools-core/dist/backend.js
index 3eecd30..f767a14 100644
--- a/node_modules/react-devtools-core/dist/backend.js
+++ b/node_modules/react-devtools-core/dist/backend.js
@@ -14569,9 +14569,9 @@ function hook_arrayLikeToArray(arr, len) { if (len == null || len > arr.length)
  */
 
 function installHook(target) {
-  if (target.hasOwnProperty('__REACT_DEVTOOLS_GLOBAL_HOOK__')) {
-    return null;
-  }
+  // if (target.hasOwnProperty('__REACT_DEVTOOLS_GLOBAL_HOOK__')) {
+  //   return null;
+  // }
 
   var targetConsole = console;
   var targetConsoleMethods = {};
